Makefile

```diff
@@ -92,10 +92,8 @@ help:
 	@echo "    test-all:           Run all tests"
 	@echo "    test-data:          Generate test data"
 	@echo "    test-module:        Run all module unit-tests"
-	@echo "    test-runtime:       Run all osbuild pipeline tests"
```

Comment:
Totally naive question: why was it called `runtime` and why is `run` more correct?

Reply:
7a745e4084fb62a104a70c426563b62b3e2df1e2
@gicmo ?

Reply:
I refer to the commit that introduced it: ff8cd76def872e253b5d31485e43ffc1f7ed45bc: `runtime`, `module` are the long names for `run` and `mod` (which are the directory names). The targets renamed with this commit 55da864e0a6f5b80190ac654118d69a13239c8bb but the documentation was not updated. 
So run is now more correct because that is the target. And `test-module` is outdated too.

Reply:
Nope, sorry, `test-module` is got renamed again from `test-mod` (94cc2cb5a7f072efb4e0d9238f737293e37bc59d) and now we have inconsistency.

Reply:
ah, got it! in that case: do for it! but would be great if you could add the explanation to  the commit message

Reply:
> would be great if you could add the explanation to the commit message

Linked to the commit that renamed the target in the commit message now.